### PR TITLE
Create governance md file

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Review by taylor — APPROVED** [review]
lgtm as first step to split this out from the charter

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Review by jeffsaelens — APPROVED** [review]
LGTM

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+## Governance
```
Comment (iawells) [inline GOVERNANCE.md:1]: You need one less hash in your titles at some point - your nesting will fail superlint.

**Review by iawells — APPROVED** [review]
lgtm, syntax fix can come later

**Review by Smitholi67 — APPROVED** [review]
lgtm

**Pull request merged**